docs/meetings/hpc/2023-12-06.md

```diff
@@ -0,0 +1,78 @@
+---
+tags: meeting, notes
+---
+
+# JupyterHub HPC Meeting - December 2023
+
+- **Date:** 2023-12-06
+- **Time:** 8:30 AM PST
+  - **Your timezone:** https://arewemeetingyet.com/Los%20Angeles/2023-12-06/08:30/JupyterHub-HPC
+- **Where you can find these and past meeting notes:** https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings/hpc/ 
+- **Calendar for future meetings:** https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings/
+- **Jupyter Code of Conduct:** https://jupyter.org/governance/conduct/code_of_conduct.html
+
+## Welcome to the Meeting
+
+Hello! If you are joining the team video meeting, sign in below so we know who was here. Roll call:
+
+- **name** / **institution** / **GitHub handle**
+- Rollin Thomas / NERSC / @rcthomas
+- Matt Henderson / LBNL / @mlhenderson
+- Michael Milligan / MSI @ UMN / @mbmilligan
+- Ben Winjum / UCLA / @benjum
+- Félix-Antoine Fortin / Université Laval / @cmd-ntrf
+
+## Quick updates
+
+Things you have been up to, questions you have, or developments you think people should know about. This is also a chance to suggest a future presentation if you've got work currently in progress you might want to share. Please add yourself, and if you do not have an update to share, you can pass.
+
+- **Name:** Your update
+- 
+
+## Agenda items
+
+Let's collect all potential agenda items here before the start of the meeting. We will then attempt to create a coherent agenda that fits in the 60m meeting slot. If there are similar items try and group them together.
+
+- **All:** Standing project items:
+    - Batchspawner check-in, issues and PRs
+        - release ready as of last month - still puzzled by new release procedures
+        - will give it another try this week, otherwise will release manually to get changes published
+    - Wrapspawner check-in, issues and PRs
+    - Discourse
+- Workshop/conference news:
+    - KubeCon NA Chicago Nov. 2023
+        - November 6-9
+        - HPC track
+        - Report?
+    - SC23
+        - HPPSS High Performance Python for Science at Scale
+            - 4 demo talks
+            - Panel: NVIDIA RAPIDS, Siu Lam (Numba), Travis Oliphant, Torsten Hofler, Fernanda Foertter 
+        - Interactive and Urgent HPC BOF
+        - Jupyter in the background
+        - Magic Castle Tutorial went well, maybe 17 participants
+        - Rollin talked to Mathworks about their Matlab integration and we'll be testing it out
+    - Others?
+- Demo - JupyterLab extension for Shutdown Alerts
```

Comment:
Is there a link to this?

Reply:
@mlhenderson Have you put this in a public repo yet?